engine/src/flutter/impeller/base/version.h

```diff
@@ -29,11 +29,11 @@ struct Version {
   static std::optional<Version> FromVector(const std::vector<size_t>& version);
 
   constexpr bool IsAtLeast(const Version& other) const {
-    return std::tie(major_version, minor_version, patch_version) >=
-           std::tie(other.major_version, other.minor_version,
-                    other.patch_version);
+    return *this >= other;
   }
 
+  constexpr auto operator<=>(const Version&) const = default;
```

Comment:
Maybe add a comment to where the fields are defined that their order matters so that the default comparison operator does the right thing?

Reply:
That's just the default behavior. Do we want to restate that everytime? Fine either way.